### ISSUE TITLE
Source & dest switched around in move_look.py?

### ISSUE THREAD

**Issue by Eric-at-Aftershock** [llooker/python_api_samples#1]
In the snippet below, from `move_look.py`, the parameters are all for the source but are then used for `dest_looker`, then the parameters are all for the destination but are then used for `source_looker`.  Isn't that switched around?

> my_host = params['hosts'][source_name]['host']
my_secret = params['hosts'][source_name]['secret']
my_token = params['hosts'][source_name]['token']
>
>dest_looker = LookerApi(host=my_host, token=my_token, secret = my_secret)
>
>my_host = params['hosts'][destination_name]['host']
my_secret = params['hosts'][destination_name]['secret']
my_token = params['hosts'][destination_name]['token']
>
>source_looker = LookerApi(host=my_host, token=my_token, secret = my_secret)

I think it should be like this:

> my_host = params['hosts'][source_name]['host']
my_secret = params['hosts'][source_name]['secret']
my_token = params['hosts'][source_name]['token']
>
>source_looker = LookerApi(host=my_host, token=my_token, secret = my_secret)
>
>my_host = params['hosts'][destination_name]['host']
my_secret = params['hosts'][destination_name]['secret']
my_token = params['hosts'][destination_name]['token']
>
>dest_looker = LookerApi(host=my_host, token=my_token, secret = my_secret)

In other words, the `params...[source_name]` should be used to make `source_looker`, and the `params...[destination_name]` should be used to make `dest_looker`.
